src/components/interface/AudioPlay.jsx

```diff
@@ -33,20 +103,27 @@ class AudioPlayer extends React.Component {
         <div style={{ alignSelf: 'center', flex: 3 }}>
           {fileName}
         </div>
-        <div style={{ alignSelf: 'center', flex: 2 }}>
+        <div style={{ alignSelf: 'center', flex: 0 }}>
           {time}
         </div>
-        <div>
-          <PlayCircleOutlineIcon style={{ float: 'right', marginLeft: '25px' }} color="primary" />
+        <div style={{flex: 0}}>
+          {
+            audioFile == undefined 
+              ? <Button disabled={true} type='transparent' onClick={this.handleClick}>
+                <PlayCircleOutlineIcon style={{ float: 'right', marginLeft: '25px' }} color="primary" />
+              </Button>
+              : <Button type='transparent' onClick={this.handleClick}>
+                <PlayCircleOutlineIcon style={{ float: 'right', marginLeft: '25px' }} color="primary" />
```

Comment:
It looks like these 2 may be consolidated into 1 statement, A lot of the code here repeats. Maybe we could use a single declaration but just use some conditions to outline which parameters should be set and which ones shouldn't inline? :)

Reply:
Done